pyof/utils.py

```diff
@@ -0,0 +1,37 @@
+"""General Unpack utils for python-openflow."""
+import pyof.v0x01.common.header
+import pyof.v0x01.common.utils
+import pyof.v0x04.common.header
+import pyof.v0x04.common.utils
```

Comment:
- "unused import statements like these", That's because they are not unused, as I said before.
-This is not a special case. Unless we make the imports inside the __init__ s, those imports are needed.